libs/wingsdk/test/target-sim/secret.test.ts

```diff
@@ -15,11 +25,51 @@ test("create a secret", async () => {
     },
     path: "root/my_secret",
     props: {
-      secretValue: '{"secret":"value"}',
+      name: "wing-sim-test-my-secret",
     },
     type: "wingsdk.cloud.Secret",
   });
   await s.stop();
 
   expect(app.snapshot()).toMatchSnapshot();
 });
+
+test("can get the secret value", async () => {
+  // GIVEN
+  const app = new SimApp();
+  cloud.Secret._newSecret(app, "my_secret", {
+    name: "wing-sim-test-my-secret",
+  });
+
+  await fs.ensureFile(path.join(os.homedir(), ".wing", "secrets.json"));
+  const secretsContent = await fs.readFile(
+    path.join(os.homedir(), ".wing", "secrets.json"),
+    "utf-8"
+  );
+
+  const secrets = tryParse(secretsContent) ?? {};
+  await fs.writeFile(
+    secretsFile,
+    JSON.stringify({
+      ...secrets,
+      "wing-sim-test-my-secret": "secret-value",
+    })
+  );
+
+  const s = await app.startSimulator();
+  const secretClient = s.getResource("/my_secret") as cloud.ISecretClient;
+
+  // THEN
+  const secretValue = await secretClient.value();
+  expect(secretValue).toBe("secret-value");
+  await s.stop();
+});
+
+function tryParse(content: string): string | undefined {
```

Comment:
Node should really just have this built-in 😄

Reply:
Can wing have this? Can I add it to `Json`?

Reply:
@jogold Sounds reasonable to me :) let's add it!